kernel/kernel-impl/src/main/java/org/sakaiproject/db/impl/BasicSqlService.java

```diff
@@ -419,7 +420,7 @@ public boolean transact(Runnable callback, String tag)
 	}
 
 	/** Used to work with dates in GMT in the db. */
```

Comment:
@ottenhoff, as comment states, can you think of anything that requires GMT in the db?  Would changing the timezone not be a good idea here?  Although it fixed one of my test cases, I wouldn't want to break something else that depends on it being GMT? e.g. doesn't Samigo depend on GMT?  but I don't know if Samigo would be affected by this change.

Reply:
Sakai depends on serializing string dates to and from GMT/UTC changing this would likely have consequences.

Reply:
> Sakai depends on serializing string dates to and from GMT/UTC changing this would likely have consequences.

Please show one place